yml/dp02_dc2.yaml

```diff
@@ -7090,692 +7090,692 @@ tables:
     "@id": "#DiaObject.decl"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Declination coordinate of the position of the diaObject at time radecTai
   - name: diaObjectId
     "@id": "#DiaObject.diaObjectId"
     datatype: long
     mysql:datatype: BIGINT
-    description:
+    description: Unique id.
   - name: gPSFluxChi2
     "@id": "#DiaObject.gPSFluxChi2"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Chi^2 statistic for the scatter of gPSFlux around gPSFluxMean
   - name: gPSFluxErrMean
     "@id": "#DiaObject.gPSFluxErrMean"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Mean of the diaSource PSF flux errors
```

Comment:
For all these things that are "mean of diaSource X": should we expand that to e.g. "Mean of the individual diaSource X" or something? Someone not entirely familiar with diaSource vs. diaObject might need a little more explicitness, maybe?